### PR TITLE
Update Service Instance

### PR DESCRIPTION
**Pull request by xxx7xxxx**

### CODE REVIEW

File: v1alpha1/meshmodel.proto [hunk 1]
```diff
@@ -174,30 +174,33 @@ message Ingress {
 
 // ServiceInstance is the runnable entity of a Mesh Service.
 message ServiceInstance {
+	// registryName is the name of registry.
```
Comment (benja-wu) [inline v1alpha1/meshmodel.proto:177]: If this requires forward compatibility, this field should be added as the last field in PB which is the eighth field. 
Ignore this suggestion if no such require needed.

Comment (xxx7xxxx) [reply v1alpha1/meshmodel.proto:177]: We will publish a new version of Easegress, EaseMesh, and EaseMeshAPI including this or and https://github.com/megaease/easegress/pull/62 . The RESTful API adjustment is incompatible, but it must be adjusted to the right way. So we do it at one time.

Comment (benja-wu) [reply v1alpha1/meshmodel.proto:177]: Ok, I got it.

**Review by benja-wu — APPROVED** [review]
LGTM

**Review by localvar — APPROVED** [review]

**Pull request merged**